thrift0.13/src/main/java/com/linecorp/armeria/internal/common/thrift/ThriftFunction.java

```diff
@@ -346,14 +347,16 @@ public boolean setException(TBase<?, ?> result, Throwable cause) {
         final String ifaceTypeName = typeName(type, funcClass, methodName, "Iface");
         try {
             final Class<?> ifaceType = Class.forName(ifaceTypeName, false, funcClass.getClassLoader());
-            for (Method m : ifaceType.getDeclaredMethods()) {
-                if (!m.getName().equals(methodName)) {
-                    continue;
+            final Method[] declaredMethods = ifaceType.getDeclaredMethods();
```

Comment:
nit: How about inlining this variable back to the `for` loop?

Reply:
ok